### DISCUSSION TITLE
Suggestion to remove the prominent message showing there is nothing on the ground in every room.

### DISCUSSION THREAD

**Discussion by ArmoredArtist** [GoMudEngine/GoMud#98] [Ideas]
To further clear up the screen clutter there is no reason to show there is nothing on the ground under each room description when you walk in.  That is apparent from there not being an item shown on the ground without the need for the message. It is a repeating message that causes spaces to appear in every room you enter to display it. I suggest that it should only show a message if something is on the ground.

**Comment by ArmoredArtist** [comment]
Also this should apply to the "Nobody else is here" message. If someone else is in the room it should show it. If nobody is in the room, you know already if it does not show it.

**Comment by Volte6** [comment]
This will be changed in the next update

**Comment by Volte6** [comment]
Addressed in https://github.com/Volte6/GoMud/pull/101

Closing.
